### PR TITLE
docs(api): redo request API for java

### PR DESCRIPTION
**Pull request by yury-s**

### CODE REVIEW

File: docs/src/api/class-requestoptions.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
+# class: RequestOptions
+* langs: java
+
+The [RequestOptions] allows to create form data to be sent via [APIRequestContext].
```
Comment (dgozman) [inline docs/src/api/class-requestoptions.md:4]: This sentence seems wrong.

File: docs/src/api/class-requestoptions.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+this request body. If this parameter is specified `content-type` header will be set to `multipart/form-data`
+unless explicitly provided.
+
+### param: RequestOptions.setMultipart.form
+- `form` <[FormData]>
+
+Form data to be serialized as html form using `multipart/form-data` encoding and sent as
+this request body.
+
+## method: RequestOptions.setQueryParam
```
Comment (dgozman) [inline docs/src/api/class-requestoptions.md:83]: addQueryParam?

File: docs/src/api/class-formdata.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    .set("age", 30);
+page.request().post("http://localhost/submit", RequestOptions.create().setForm(form));
+```
+
+## method: FormData.create
+- returns: <[FormData]>
+
+Creates new instance of [FormData].
+
+## method: FormData.set
```
Comment (dgozman) [inline docs/src/api/class-formdata.md:21]: Can you do the same name multiple times? If so, I'd call this `add`.

File: docs/src/api/params.md [hunk 4]
```diff
@@ -303,28 +303,39 @@ Emulates consistent window screen size available inside web page via `window.scr
 
 Target URL.
 
-## fetch-option-params
+## js-python-fetch-option-params
```
Comment (dgozman) [inline docs/src/api/params.md:306]: I'd rename this and similar back to `fetch-option-param` since it's the default style.

**Review by dgozman — APPROVED** [review]

File: docs/src/api/class-requestoptions.md (hunk 2 above)
Comment (yury-s) [reply docs/src/api/class-requestoptions.md:83]: Currently it behaves as `set` we can add `add` method later if there is demand.

File: docs/src/api/class-formdata.md (hunk 3 above)
Comment (yury-s) [reply docs/src/api/class-formdata.md:21]: Same here, we can add `append` if need be.

File: docs/src/api/params.md (hunk 4 above)
Comment (yury-s) [reply docs/src/api/params.md:306]: They are now available only in js and python and for such cases we include languages into the prefix in other places, so this is consistent with existing approach.

**Pull request merged**